releng/sap-kyma-runtime/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Docker descriptor for XSK
+# License - https://github.com/SAP/xsk/blob/main/LICENSE
+
+FROM tomcat:9.0.56-jre11-openjdk-slim AS base
+
+COPY target/ROOT.war $CATALINA_HOME/webapps/
+
+# Copy application folder
+ARG APP_PATH
+COPY $APP_PATH /usr/local/tomcat/target/dirigible/repository/root/registry/public/
```

Comment:
I think this won't be needed, as the goal here is to provide a `runtime only` image that can be used as a base image in custom `Dockerfile` to build an immutable application. Hence it seems that the `APP_PATH` arg is not needed here.

Reply:
Ok. I will remove this.